update-remote.sh

```diff
@@ -39,7 +39,7 @@ for repo in "${repos[@]}"; do
 			href="#${href,,}"
 			echo $'-\t[`'"$tag"'`]('"$href"')'
 		done
-		docker run -i --rm repo-info:tag-details "${tags[@]}"
+		docker run -i --rm repo-info:remote -- "${tags[@]}"
```

Comment:
I didn't convert this file to use a single `docker run` instance yet since this PR was already big enough as-is. :+1: